flink-kubernetes/src/main/java/org/apache/flink/kubernetes/highavailability/KubernetesStateHandleStore.java

```diff
@@ -660,6 +661,12 @@ private Optional<KubernetesConfigMap> addEntry(
                                 "Unable to remove the marked as deleting entry.");
                     }
                 } else {
+                    // It could happen the fabric8 k8s client retries a transaction that has
```

Comment:
```suggestion
                    // It could happen that the k8s client retries a transaction that has
```
nit: no need to mention the type of k8s client here to avoid the risk of having out-dated comments if we decide to change the internally used client. `fabric8` also doesn't add any value here